spring-boot-project/spring-boot-testcontainers/src/main/resources/META-INF/spring.factories

```diff
@@ -38,4 +38,5 @@ org.springframework.boot.testcontainers.service.connection.r2dbc.PostgresR2dbcCo
 org.springframework.boot.testcontainers.service.connection.r2dbc.SqlServerR2dbcContainerConnectionDetailsFactory,\
 org.springframework.boot.testcontainers.service.connection.redis.RedisContainerConnectionDetailsFactory,\
 org.springframework.boot.testcontainers.service.connection.redpanda.RedpandaContainerConnectionDetailsFactory,\
-org.springframework.boot.testcontainers.service.connection.zipkin.ZipkinContainerConnectionDetailsFactory
+org.springframework.boot.testcontainers.service.connection.zipkin.ZipkinContainerConnectionDetailsFactory,\
+org.springframework.boot.testcontainers.service.connection.hazelcast.HazelcastContainerConnectionDetailsFactory
```

Comment:
The entries in that file should be sorted alphabetically.

Reply:
Sorted by qualified name or just class name?